pkg/apis/logging/v1/clusterlogging_types.go

```diff
@@ -210,6 +215,147 @@ type CuratorSpec struct {
 	Schedule string `json:"schedule"`
 }
 
+// OutputConfSpec represents the output configuration for tuning parameters per
+// output type, e.g. Fluentd. The tuning parameters are applied for all outputs
+// of the same type. Currently supported: `fluentd`.
+type OutputConfSpec struct {
+	Fluentd *FluentdOutputSpec `json:"fluentd,omitempty"`
+}
+
+// FluentdOutputSpec represents the configuration for outputs of type fluentd.
+// It supports overriding settings for fluentd buffer configurations. The settings
+// apply to all buffer configurations of each fluentd output.
```

Comment:
Take over comment from #595 [comment](https://github.com/openshift/cluster-logging-operator/pull/595#discussion_r455066469):
> Nit. I think this is more obvious in the CRD generation but I think this particular comment should be less specific should it not? These settings are for overriding Fluent Output configuration . The included spec are specifically for buffer

Reply:
Possibly, I was getting paranoid and probably over-explaining things when I wrote that. Feel free improve the wording :)